client/main.lua

```diff
@@ -315,8 +319,10 @@ RegisterNetEvent('QBCore:Client:OnPlayerUnload', function()
     deletePeds()
 end)
 
-RegisterNetEvent('qb-cityhall:client:getIds', function()
-    TriggerServerEvent('qb-cityhall:server:getIDs')
+RegisterNetEvent('qbx-core:server:createCharacter', function()
```

Comment:
Isn't this a server side event? Also server would never have source since it isn't coming from a player